index.js

```diff
@@ -34,9 +38,10 @@ module.exports = function() {
 
       // Unfortunately rework unconditionally splits selectors by comma, so our
       // :matches(...) selector gets munged.
-      var actualSelectors = rule.selectors.join(',').replace(/:matches\(.*\)/g, function(substr) {
+      var actualSelectors = rule.selectors.join(',').replace(/:matches\(.*?\)/g, function(substr) {
```

Comment:
Agreed. This is definitely something that could be solved in a more central, standardized location -- whether through a node library, a rework-plugin, or in rework itself.

Though, I'm inclined to think this is best solved uniformly at the rework level since it can already handle `:not`, `:nth-child`, etc. I think its wise for the long term goal to be adding `:matches` support (since it's part of the CSS4 spec). Though, I also think that being able to handle `:some-pseudo-selector(:with, :a, :comma, :delimited, :list)` is a rework concern. Having it end up as five different selectors is just silly :smirk_cat: .

Perhaps if I get some time this weekend I might have a dive into rework and the css module and see how complex it'd be to implement.